### DISCUSSION TITLE
Segregate the builder from the actual Vaultwarden LXC container

### DISCUSSION THREAD

**Discussion by circa1741** [tteck/Proxmox#114] [Ideas]
Instead of a single LXC container with all the require components to build and to run Vaulwarden, consider using a separate LXC container with the required components **to build** Vaultwarden from source as a .deb package file.

Then, the actual Vaultwarden LXC container downloads and installs the built .deb file and other necessary components **to run** Vaultwarden.

**Comment by tteck** [comment]
Are you volunteering for the builder position?  😉

**Comment by circa1741** [comment]
Have not done Proxmox scripting. I will try to figure out the existing scripts and see if I can slice and dice it. Don't hold your breath...
